modular_skyrat/master_files/code/modules/power/singularity/containment_field.dm

```diff
@@ -0,0 +1,2 @@
+/obj/machinery/field/containment/singularity_act()
+	return
```

Comment:
is this the machine that generates the field? or the field itself?

Reply:
The field itself